parkingspaces/src/main/java/com/baspiotr/parkingspaces/configuration/DatabaseConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package com.baspiotr.parkingspaces.configuration;
+
+import com.baspiotr.parkingspaces.domain.model.Parking;
+import com.baspiotr.parkingspaces.domain.model.Role;
+import com.baspiotr.parkingspaces.domain.model.User;
+import com.baspiotr.parkingspaces.domain.repository.ParkingRepository;
+import com.baspiotr.parkingspaces.domain.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.transaction.Transactional;
+import java.util.Arrays;
+
+@Configuration
+public class DatabaseConfiguration {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    ParkingRepository parkingRepository;
+
+    @PostConstruct
+    @Transactional
+    public void initDB() {
+        User driver = User.builder().firstName("Piotr").lastName("Basinski").role(Role.DRIVER_REGULAR).build();
```

Comment:
I have a proposal - let's delete it from here and do it in tests.
You can create such tests that 
1. prepare data in DB
2. call rest controllers using mockMvc https://docs.spring.io/spring-security/site/docs/current/reference/html/test-mockmvc.html and check if there was expected result

in real app you would need to implement logic for adding new parking etc, but it is out of scope so just stick to tests only.

you need to have a few tests, like 10 or more, doing different scenarios